pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -635,7 +635,7 @@ def test_add_empty_datetimeindex(self, offset_types, tz_naive_fixture):
             result = offset_s + dta
         tm.assert_equal(result, dta)
 
-    def test_pickle_v0_15_2(self, datapath):
+    def test_pickle_v1_1_0(self, datapath):
```

Comment:
can you execute the code on 1.0.x instead of 1.1 (as it hasn't been released yet). alternatively you can remove this test entirely and simply commit generate pickles for 1.0.x via this: https://github.com/pandas-dev/pandas/blob/master/pandas/tests/io/generate_legacy_storage_files.py (you will likey to have to update to generate the case that is below). This will be more robust.

Reply:
I regenerated the pickle file in v1.0.5. Also I notice in Python 3.8 they changed `pickle.DEFAULT_PROTOCOL` to `4`, so to avoid ambiguity I explicitly set `protocol=4`. Can you review?

I didn't quite get what you mean by "remove this test entirely and ...". If I remove this test, that legacy file would no longer be needed and I think we can remove that too. I didn't want to do this though because I think this test does add value in testing the base cases of pickling the various types of offsets.

Reply:
look at the pointer i gave
that file pickles everything for a given version 

you may need to add some more cases to the script

Reply:
Have generated the new pickle file as suggested. However there is an issue with pickle protocol. I will comment below for discussion since it's a separate issue.